### PR TITLE
Add Eldarya and all its websites

### PR DESCRIPTION
**Pull request by NatoBoram**
Basically #16974, thanks to https://github.com/EFForg/https-everywhere/issues/16974#issuecomment-433370040. I've also written a [patch](https://github.com/EFForg/https-everywhere/issues/16974#issuecomment-432079522) if you guys prefer that.

### CODE REVIEW

File: src/chrome/content/rules/Eldarya.com.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+	h: http redirect
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="Eldarya.com">
+	<target host="eldarya.com" />
+	<target host="www.eldarya.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Eldarya.com.xml]: You should have a look at https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#enumerating-subdomains

**Comment by pipboy96** [comment]
@NatoBoram Please fix the issues pointed out by @J0WI.

**Comment by NatoBoram** [comment]
Hm, it's been a year. I no longer have the fork on my system, nor do I have said system anymore, so I'll let you guys do it. The list of domains and subdomains is enumerated [here](https://github.com/EFForg/https-everywhere/issues/16974). I don't know if they added some translations, but there's a banner at the top of the site with relevant country flags.

**Comment by pipboy96** [comment]
@NatoBoram Thanks for clarifying!

**Pull request closed without merging**